clang/lib/Driver/Driver.cpp

```diff
@@ -6677,3 +6678,136 @@ llvm::Error driver::expandResponseFiles(SmallVectorImpl<const char *> &Args,
 
   return llvm::Error::success();
 }
+
+namespace {
+
+const char *GetStableCStr(std::set<std::string> &SavedStrings, StringRef S) {
+  return SavedStrings.insert(std::string(S)).first->c_str();
+}
+
+/// applyOneOverrideOption - Apply a list of edits to the input argument lists.
```

Comment:
https://llvm.org/docs/CodingStandards.html

Don’t duplicate the documentation comment in the header file and in the implementation file.

Reply:
I will move it to the header file.

Reply:
fixed these two and the include you pointed out.